src/MailpitCommands.ts

```diff
@@ -127,12 +112,7 @@ class MailpitCommands {
 				},
 				auth: this.auth,
 			})
-			.then((response) => {
-				if (response.status !== 200) {
-					throw new Error(`Failed to get messages. Status: ${response.status}, Body: ${response.body}`);
-				}
-				return response.body as MessagesSummary;
-			});
+			.its('body');
```

Comment:
can we not change the return here ?
```ts
then((response) => {
	return response.body as MessagesSummary;
});
```

Reply:
.its('body') is returning the body which is casted to MessageType but if it is wanted I can change it.

https://docs.cypress.io/api/commands/its > Get a property's value on the previously yielded subject.

```suggestion
			.then(response => response.body as MessagesSummary);
```

I think both working the same.